src/components/shared/formItems/ProjectDuration.tsx

```diff
@@ -35,6 +35,11 @@ export default function ProjectDuration({
               onChange={checked => {
                 setShowDurationInput(checked)
                 onValueChange(checked ? '30' : '0')
+
+                // If toggling off Funding Cycle Duration when set to "one-time", revert it to "recurring"
+                if (!isRecurring && !checked) {
+                  onToggleRecurring()
```

Comment:
Noblocking nit: It might be clearer what's happening here if we switched `onToggleRecurring` to `setIsRecurring` and explicitly set the value. We can do toggle behavior in other places with `setIsRecurring(!isReucrring)`.